telegram/ext/_precheckoutqueryhandler.py

```diff
@@ -75,15 +75,12 @@ async def callback(update: Update, context: CallbackContext)
     def __init__(
         self,
         callback: HandlerCallback[Update, CCT, RT],
-        pattern: Optional[Union[str, Pattern[str]]] = None,
         block: DVType[bool] = DEFAULT_TRUE,
+        pattern: Optional[Pattern[str]] = None,
```

Comment:
```suggestion
        pattern: Optional[Union[str, Pattern[str]]] = None,
```

accepting a string as input is okay, my point in https://github.com/python-telegram-bot/python-telegram-bot/pull/4005#discussion_r1427212996 was just that we can convert strings to patterns on init already :) sorry if that was unclear.